src/Cardiompp.Domain/Entities/CardiomppAgent.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace Cardiompp.Domain.Entities
+{
+    public class CardiomppAgent
+    {
+        public CardiomppAgent(
```

Comment:
O padrão de identação do c# utiliza as chaves `{` em baixo das desclarações das função ou blocos lógicos.
Por que não fazer o mesmo com os parênteses `(` ?
```suggestion
        public CardiomppAgent
        (
```

Reply:
Feito.

![image](https://user-images.githubusercontent.com/12301754/80872748-5654e080-8c8a-11ea-9cec-d8f5955082b2.png)